### PR TITLE
Removed all modding-related content and moved it to RTD

### PR DESCRIPTION
**Pull request by emma-miler**
Removed all modding-related content and moved it to [ReadTheDocs](https://r2northstar.readthedocs.io).
Already checked for backlinks and updated them.

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Blocking until [R2Northstar/ModdingDocs](https://github.com/R2Northstar/ModdingDocs) gets a proper README :3

**Comment by emma-miler** [comment]
It has a README now, not sure if you think it's proper :P

**Comment by GeckoEidechse** [comment]
You're still missing a license though :P

**Comment by emma-miler** [comment]
Good point, i'll copy it over from this repo and fix the merge conflicts

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -35,29 +35,6 @@
 
 * [Modding](modding/README.md)
   * [Getting Started](modding/getting-started.md)
```
Comment (GeckoEidechse) [inline docs/SUMMARY.md]: You missed a line :3

```suggestion
```

File: docs/modding/localisation_files.md [hunk 2]
```diff
@@ -1,12 +0,0 @@
...
-Northstar adds new strings to the game which can be localised to match the language you are using on your Titanfall 2 installation.
-
-## Files to translate
-
-There are the two files that need to be translated, use the English version as a base. After you finish make sure that the files are encoded in `UTF-16 LE` before opening a Pull Request.
-
-1. [Northstar.Client/mod/resource/northstar_client_localisation_english.txt](https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Client/mod/resource/northstar_client_localisation_english.txt)
-2. [Northstar.Custom/mod/resource/northstar_custom_english.txt](https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.Custom/mod/resource/northstar_custom_english.txt)
-
-To test your modifications go to `Origin (My games library) -> Titanfall 2 (right click) -> Game Properties -> Advanced Launch Options` and select the language you modified from the dropdown.
```
Comment (GeckoEidechse) [inline docs/modding/localisation_files.md:12]: Do we have localisation instructions in the new modding docs? Couldn't find any :c

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]

**Comment by Alystrasz** [comment]
@emma-miler please finish your PR :smiley: 

(asking for a friend btw)
Capture d’écran de 2022-03-23 14-31-19

**Comment by emma-miler** [comment]
Updated

**Review by GeckoEidechse — APPROVED** [review]

**Pull request merged**